packages/@lwc/engine-server/src/__tests__/fixtures/attribute-component-global-html/expected.html

```diff
@@ -1,6 +1,6 @@
 <x-attribute-component-global-html>
   <template shadowrootmode="open">
-    <x-child accesskey="A" class="bar foo" contenteditable="true" data-test="test" draggable="true" hidden id="foo" itemprop="foo" spellcheck="true" style="color: red; background: blue;" tabindex="-1">
+    <x-child accesskey="A" class="bar foo" data-test="test" draggable="true" hidden id="foo" spellcheck="true" style="color: red; background: blue;" tabindex="-1">
```

Comment:
Are we going to break anyone by no longer reflecting contenteditable/itemprop?

Reply:
I don't think so, this is the current behavior of engine-dom

Reply:
I don't see any `integration-karma` tests for this. Let's add a hydration test to confirm this behavior (will double as a test that `engine-server` and `engine-dom` output the same thing).

Reply:
We already had tests that test reflection so I added a test for `contenteditable` to show that `engine-server` and `engine-dom` should both not reflect that attribute.